src/libs/actions/QuickActionNavigation.ts

```diff
@@ -49,7 +53,20 @@ function navigateToQuickAction(isValidReport: boolean, quickAction: QuickAction,
             break;
         case CONST.QUICK_ACTIONS.TRACK_MANUAL:
         case CONST.QUICK_ACTIONS.TRACK_SCAN:
+            selectOption(() => startMoneyRequest(CONST.IOU.TYPE.TRACK, reportID, requestType, true), false);
+            break;
+        case CONST.QUICK_ACTIONS.REQUEST_DISTANCE:
+            if (isManualDistanceTrackingEnabled) {
+                selectOption(() => startDistanceRequest(CONST.IOU.TYPE.SUBMIT, reportID, requestType, true), false);
+                return;
+            }
+            selectOption(() => startMoneyRequest(CONST.IOU.TYPE.SUBMIT, reportID, requestType, true), true);
+            break;
         case CONST.QUICK_ACTIONS.TRACK_DISTANCE:
+            if (isManualDistanceTrackingEnabled) {
+                selectOption(() => startDistanceRequest(CONST.IOU.TYPE.TRACK, reportID, requestType, true), false);
+                return;
+            }
```

Comment:
I think it makes more sense to create new constants like `CONST.QUICK_ACTIONS.REQUEST_DISTANCE_MANUAL` and `CONST.QUICK_ACTIONS.REQUEST_DISTANCE_MAP` instead of passing in the feature flag.